x-pack/plugin/esql/src/main/generated-src/org/elasticsearch/xpack/esql/expression/predicate/operator/comparison/InBytesRefEvaluator.java

```diff
@@ -0,0 +1,185 @@
+                if (lhsVector == null) {
+                    return eval(page.getPositionCount(), lhsBlock, rhsBlocks);
+                }
+                BytesRefVector[] rhsVectors = new BytesRefVector[rhs.length];
+                for (int i = 0; i < rhsBlocks.length; i++) {
+                    rhsVectors[i] = rhsBlocks[i].asVector();
+                    if (rhsVectors[i] == null) {
+                        return eval(page.getPositionCount(), lhsBlock, rhsBlocks);
+                    }
+                }
+                return eval(page.getPositionCount(), lhsVector, rhsVectors);
+            }
+        }
+    }
+
+    public BooleanBlock eval(int positionCount, BytesRefBlock lhsBlock, BytesRefBlock[] rhsBlocks) {
+        try (BooleanBlock.Builder result = driverContext.blockFactory().newBooleanBlockBuilder(positionCount)) {
+            BytesRef[] rhsValues = new BytesRef[rhs.length];
+            BytesRef lhsScratch = new BytesRef();
+            BytesRef[] rhsScratch = new BytesRef[rhs.length];
+            for (int i = 0; i < rhs.length; i++) {
+                rhsScratch[i] = new BytesRef();
+            }
+            BitSet nulls = new BitSet(rhs.length);
+            BitSet mvs = new BitSet(rhs.length);
+            for (int p = 0; p < positionCount; p++) {
+                if (lhsBlock.isNull(p)) {
+                    result.appendNull();
+                    continue;
+                }
+                if (lhsBlock.getValueCount(p) != 1) {
+                    if (lhsBlock.getValueCount(p) > 1) {
+                        warnings.registerException(new IllegalArgumentException("single-value function encountered multi-value"));
+                    }
+                    result.appendNull();
+                    continue;
+                }
+                // unpack rhsBlocks into rhsValues
+                nulls.clear();
+                mvs.clear();
+                for (int i = 0; i < rhsBlocks.length; i++) {
+                    if (rhsBlocks[i].isNull(p)) {
+                        nulls.set(i);
+                        continue;
+                    }
+                    if (rhsBlocks[i].getValueCount(p) > 1) {
+                        mvs.set(i);
+                        warnings.registerException(new IllegalArgumentException("single-value function encountered multi-value"));
+                        continue;
+                    }
+                    int o = rhsBlocks[i].getFirstValueIndex(p);
+                    rhsValues[i] = rhsBlocks[i].getBytesRef(o, rhsScratch[i]);
+                }
+                if (nulls.cardinality() == rhsBlocks.length || mvs.cardinality() == rhsBlocks.length) {
+                    result.appendNull();
+                    continue;
+                }
+                try {
+                    In.process(result, nulls, mvs, lhsBlock.getBytesRef(lhsBlock.getFirstValueIndex(p), lhsScratch), rhsValues);
+                } catch (IllegalArgumentException e) {
```

Comment:
Are we throwing this one intentionally? I think maybe we can just not do the try/catch here because we really shouldn't be throwing this. I think.

Reply:
> Are we throwing this one intentionally? I think maybe we can just not do the try/catch here because we really shouldn't be throwing this. I think.

I'll remove the two try/catch, seems like not needed and I don't recall why they are there in the first place, perhaps the `process` method changed.